src/explore-education-statistics-frontend/src/services/googleAnalyticsService.ts

```diff
@@ -64,6 +64,9 @@ export function logOutboundLink(label: string, url: string) {
       label,
     },
     function hitCallback() {
+      if (newTab) {
+        window.open(url, '_blank');
```

Comment:
I think you need to return after this line otherwise it'll open in a new tab and the same tab.

Also, can you add the same logic to open the link when GA is not initialised so that links work locally without GA set up?

Reply:
Good spot, that's sorted now. I've added an if statement to ensure that the `DATA_API_BASE_URL` environment variable isn't equal to the URL that the local data API listens on: 

```typescript
  if (
    process.env.DATA_API_BASE_URL !== 'http://localhost:5000/api' &&
    !initialised
  ) {
    return;
  }
```

Reply:
I'm not really following why you've added the check for DATA_API_BASE_URL here. We want it to still open the links if GA isn't initialised, but not try to send the GA request. So I think we need to just duplicate the code that's in hitCallback here instead.